pkgs/by-name/tf/tfsort/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

This is a newer and "better" way to do it. It keeps things consistent, when users override specific things in this derivation.